src/clients/c++/perf_client/perf_client.cc

```diff
@@ -448,6 +450,25 @@ Usage(char** argv, const std::string& msg = std::string())
                    "order. Default is \"random\".",
                    18)
             << std::endl;
+  std::cerr << FormatMessage(
+                   " --shared-memory <\"system\"|\"cuda\"|\"none\">: Selects "
+                   "the type of the shared memory to use for input data. "
+                   "Default is none.",
+                   18)
+            << std::endl;
+
+  std::cerr
+      << FormatMessage(
+             " --output-shm-size: Is used to compute the size of shared "
```

Comment:
The size in bytes of the shared memory region to allocate per output tensor. Only needed when one or more of the outputs are of string type or variable shape. The value should be at least the maximum possible size (in bytes) of any of the output tensors the model can return. The client will use the following formula to calculate the total shared memory to allocate: output_shm_size * number_of_outputs * batch_size. Defaults to 100KB.

Reply:
Done